libr/core/cmd_debug.c

```diff
@@ -2729,17 +2729,17 @@ static int cmd_debug_step (RCore *core, const char *input) {
 	case 'u':
 		switch (input[2]) {
 		case 'f':
-			step_until_flag (core, input+3);
+			step_until_flag (core, input + 3);
 			break;
 		case 'i':
-			step_until_inst (core, input+3);
+			step_until_inst (core, input + 3);
 			break;
 		case 'e':
-			step_until_esil (core, input+3);
+			step_until_esil (core, input + 3);
 			break;
 		case ' ':
 			r_reg_arena_swap (core->dbg->reg, true);
-			step_until (core, r_num_math (core->num, input+2)); // XXX dupped by times
+			step_until (core, r_num_math (core->num, input + 2)); // XXX dupped by times
```

Comment:
There's not really a great place to declare it here. Anyway I'd be in favor of something larger that made command argument handling much cleaner throughout. I just don't think this PR is the right place for it.